src/webplugin/edge/manifest.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
Despite minor differences the manifest of Chrome, Edge and Firefox look very similar. Would it be possible to share certain properties (like the author, name, description, version etc.) and add platform specific changes as an extra build step?

A basic way to do this is to create a base manifest which contains all shared properties. Next you have a separate manifest for Firefox, Edge and Chrome which contain only changed or added properties. The extra build step would then merge these two manifests together and save it in the output path.